docs/spec/loops.md

```diff
@@ -1,5 +1,6 @@
 # Loops
-> **Note:** Not implemented yet.
+
+>Loops were implemented in v0.3
```

Comment:
Might be worth restating the limitations here:
- no variable loops
- no filtering
- no index syntax
- etc...